extension/src/commands/external.ts

```diff
@@ -5,6 +5,7 @@ export enum RegisteredCliCommands {
   EXPERIMENT_PUSH = 'dvc.pushExperiments',
   EXPERIMENT_REMOVE = 'dvc.removeExperiments',
   EXPERIMENT_REMOVE_QUEUE = 'dvc.removeExperimentQueue',
+  EXPERIMENT_RENAME = 'dvc.renameExperiment',
```

Comment:
[I] This should be with the view commands because it is triggered from a view i.e. `EXPERIMENT_VIEW_RENAME = 'dvc.views.experiments.renameExperiment',`